### PR TITLE
Tutorial to start a new Telestion project

### PR DESCRIPTION
**Pull request by fussel178**
### Summary 

Add a tutorial to start a new project.

### Details 

A first iteration on the steps required to set up a new Telestion project. :wink: 

### Additional information 

n/a

### CLA

- [x] I have signed the individual contributor's license agreement **and sent** it to the board of the WüSpace e. V. organization.

### CODE REVIEW

File: docs/application/tutorials/start-a-new-project.mdx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+Now, GitHub asks you some required information.
+Please select suitable user/group and give the repository a meaningful name.
+
+Next, select your repository's visibility.
+When you are satisfied with your information, press the `Create repository from template` button.
+
+<Image src="/img/new-project/github-create-repo.png" alt="GitHub's Create repository from template interface" />
+
+That's it!
```
Comment (pklaschka) [inline docs/application/tutorials/start-a-new-project.mdx:83]:
```suggestion
First, log into GitHub with your user account.

Head to our project template and press the `Use this template button`.

<Reference to="https://github.com/wuespace/telestion-project-template">
  Telestion Project Template
</Reference>

<Image src="/img/new-project/github-use-this-template.png" alt="GitHub's Use this template button" />

Now, GitHub asks you some required information.
Select a suitable user/group and give the repository a meaningful name.

Next, select your repository's visibility.
When you are satisfied with your information, press the `Create repository from template` button.

<Image src="/img/new-project/github-create-repo.png" alt="GitHub's Create repository from template interface" />

That's it!
```

File: docs/application/tutorials/start-a-new-project.mdx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+<Image src="/img/new-project/github-initialize-run-workflow.png" alt="Open popup to run the Initialize workflow in GitHub Actions" />
+
+:::tip Java package naming
+
+It is recommended to follow the 
+[Maven Central `groupId` naming conventions](https://maven.apache.org/guides/mini/guide-naming-conventions.html), 
+e.g. beginning with the company url in reverse.
+
+:::
```
Comment (pklaschka) [inline docs/application/tutorials/start-a-new-project.mdx:107]:
```suggestion
Now, you should have a new repository under your control.

Next, initialize the Telestion project.
Our template already has a GitHub Action configuration that does the initialization steps for you.

Go to the `Actions` tab and select the `Initialize` in the left sidebar.

<Image src="/img/new-project/github-actions-initialize.png" alt="GitHub Actions page with the selected Initialize action" />

Now press the `Run workflow` button in the Action list.
In the popup, enter your preferences and then press `Run workflow`.

<Image src="/img/new-project/github-initialize-run-workflow.png" alt="Open popup to run the Initialize workflow in GitHub Actions" />

:::tip Java package naming

We recommend following the
[Maven Central `groupId` naming conventions](https://maven.apache.org/guides/mini/guide-naming-conventions.html), 
e.g., beginning with the company URL in reverse.

:::
```

File: docs/application/tutorials/start-a-new-project.mdx [hunk 3]
```diff
@@ -0,0 +1,130 @@
+---
+title: Start a new Project
+# e.g., "Publishing messages to the event bus" or "Installing a dependency using npm"
+
+description: Start a new Telestion Project with the Telestion Project template
```
Comment (pklaschka) [inline docs/application/tutorials/start-a-new-project.mdx:5]:
```suggestion
title: Starting a new Project
# e.g., "Publishing messages to the event bus" or "Installing a dependency using npm"

description: Start a new Telestion Project with the Telestion Project template
```

File: docs/application/tutorials/index.mdx [hunk 4]
```diff
@@ -7,5 +7,6 @@ import { Grid, Reference } from '/components';
 ## Basics
 
 <Grid>
+	<Reference to="start-a-new-project">Start a new project</Reference>
```
Comment (pklaschka) [inline docs/application/tutorials/index.mdx:10]:
```suggestion
	<Reference to="starting-a-new-project">Starting a new project</Reference>
```

File: docs/application/tutorials/start-a-new-project.mdx [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+## Next steps
+
+<!-- Short concluding sentence: -->
+
+That's it! Now you have a new Telestion Project ready for development.
+
+<!-- Links to next steps/related articles -->
+
+Next steps: Coming soon™
```
Comment (pklaschka) [inline docs/application/tutorials/start-a-new-project.mdx:117]: Looking forward to that 😁

**Review by pklaschka — CHANGES_REQUESTED** [review]
I have one major request (see below) and a few minor improvements which I've left as inline comment suggestions 🙂

### The major request
Tutorial titles should be written in present progressive ("-ing"-form), i.e., here, "Starting a new project" instead of "Start a new project". I've left inline suggestions to change it in the text, but you'll need to rename the file 🙂 

Other than that, it looks good to me 🙂

File: docs/application/tutorials/start-a-new-project.mdx (hunk 5 above)
Comment (fussel178) [reply docs/application/tutorials/start-a-new-project.mdx:117]: :wink:

**Comment by fussel178** [comment]
Done! :D

**Review by pklaschka — APPROVED** [review]

**Comment by pklaschka** [comment]
@fussel178 Congrats on your first documentation article! 🍰

**Pull request merged**